com.unity.formats.alembic/Runtime/Scripts/Importer/Behaviours/AlembicCustomData.cs

```diff
@@ -3,15 +3,15 @@
 namespace UnityEngine.Formats.Alembic.Importer
 {
     /// <summary>
-    /// Class that stores ancillary data for the current Alembic Node.
+    /// Class that stores additional data for the current Alembic Node.
     /// </summary>
     public class AlembicCustomData : MonoBehaviour
     {
         [SerializeField]
         List<string> faceSetNames;
 
         /// <summary>
-        /// Retrieves the set of Face Set names stored in the current Alembic node.
+        /// Retrieves the set of Face Set names.
```

Comment:
I was rather suggesting:
`Retrieves the Face Set names stored in the current Alembic node.`